doc/releases/changelog-dev.md

```diff
@@ -73,6 +76,25 @@
   (with potentially negative eigenvalues) has been implemented.
   [(#5048)](https://github.com/PennyLaneAI/pennylane/pull/5048)
 
+* The decomposition of an operator created with calling `qml.ctrl` on a parametric operator (specifically `RX`, `RY`, `RZ`, `Rot`, `PhaseShift`) with a single control wire will now be the full decomposition instead of a single controlled gate. For example:
+  ```
+  >>> qml.ctrl(qml.RX(0.123, wires=1), control=0).decomposition()
+  [CRX(0.123, wires=[0, 1])]
+  ```
+  Will become:
```

Comment:
"Will become" is a bit ambiguous; I think the new behaviour + the text description you gave describes it nicely 😄 
```suggestion
```